xbmc/utils/ContentUtils.cpp

```diff
@@ -21,7 +21,7 @@ const bool HasPreferredArtType(const CFileItem& item)
                                     item.GetVideoInfoTag()->m_type == MediaTypeVideoCollection);
 }
 
-const std::string GetPreferredArtType(MediaType type)
+const std::string GetPreferredArtType(const MediaType type)
```

Comment:
```suggestion
const std::string GetPreferredArtType(const MediaType& type)
```

Reply:
+ if i see it correct returning `const std::string` is useless as it's a copy? should be fine to just return `std::string` i guess